tonic_validate/metrics/augmentation_precision_metric.py

```diff
@@ -15,6 +15,10 @@ class AugmentationPrecisionMetric(Metric):
     name: str = "augmentation_precision"
 
     def __init__(self) -> None:
+        """
+        Metric that checks whether the relevant context is in the LLM answer.
+        Returns a float between 0 and 1, where 1 is it contains all the relevant context and 0 is it contains none of the relevant context.
```

Comment:
Returns a float between 0 and 1. 1 indicates that the answer contains all of the relevant context. 0 indicates that it contains none of the relevant context.